src/request/index.ts

```diff
@@ -13,7 +13,7 @@ import {
 	SingleBatchRequestResponse,
 	SOAPHeader
 } from './types';
-
+import fetch from 'cross-fetch';
```

Comment:
how does this work in browsers? we don't want to include anything for browsers

Reply:
As per [cross-fetch documentation](https://www.npmjs.com/package/cross-fetch#install), including `cross-fetch` like this will [ponyfill](https://github.com/sindresorhus/ponyfill#ponyfill). So it won't modify original fetch api.